tests/src/test/scala/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -1035,7 +607,7 @@ class ApiGwTests
             rr.stderr should include("does not exist")
         }
         finally {
-            val deleteresult = apiDeleteExperimental(basepathOrApiName = testbasepath, expectedExitCode = DONTCARE_EXIT)
+            val deleteresult = apiDelete(basepathOrApiName = testbasepath, expectedExitCode = DONTCARE_EXIT)
```

Comment:
whiskprops is now switched out as well, however, may need to figure out what's going on in WskConfigTests.scala. I didn't see any tests for V2, so I did my best to modify the existing ones (to get around compiler errors at least). I have a feeling the tests will fail